src/handlers/staking_handlers.ts

```diff
@@ -39,14 +40,37 @@ export class StakingHandlers {
         res.status(HttpStatus.OK).send(response);
     }
     public async getStakingEpochsAsync(_req: express.Request, res: express.Response): Promise<void> {
-        const [currentEpoch, nextEpoch] = await Promise.all([
-            this._stakingDataService.getCurrentEpochAsync(),
-            this._stakingDataService.getNextEpochAsync(),
-        ]);
-        const response: StakingEpochsResponse = {
-            currentEpoch,
-            nextEpoch,
-        };
+        // optional query string to include fees
+        let isWithFees: boolean;
+        if (_req.query.withFees) {
+            if (!(_req.query.withFees === 'true' || _req.query.withFees === 'false')) {
+                res.status(HttpStatus.BAD_REQUEST).send(`Invalid value for withFees`);
+            }
+            isWithFees = _req.query.withFees === 'true';
+        } else {
+            isWithFees = false;
+        }
```

Comment:
Elsewhere we extract query params in pure functions (so we can actually type the query object)

Reply:
👍